system-contracts/bootloader/bootloader.yul

```diff
@@ -1910,6 +2033,39 @@ object "Bootloader" {
 
 
             <!-- @if BOOTLOADER_TYPE=='playground_batch' -->
+            function ethCallEvmConsturction(
```

Comment:
This is needed to return deployment bytecode if `eth_call` is used to deploy contract.
Not _directly_ related to this scope, but it was needed to make `alto` work, so I implemented it.